CHANGE_MANAGEMENT.md

```diff
@@ -0,0 +1,98 @@
+# Overview
+This is a meritocratic, consensus-based community project. Anyone with interest in the project can join the community, contribute to the project design and participate in the decision making process. Participation in these processes is completely voluntary. This document describes how that participation takes place and how to set about earning merit within the project community.
+
+Although these processes are adapted from the [OSSWatch Meritocratic Model](http://oss-watch.ac.uk/resources/meritocraticgovernancemodel), this documentation is a formalization of existing processes involving relevant stakeholders.
+
+## Roles
+### Stakeholders
+Stakeholders are community members who have a need for the tooling provided by this project. They are the most important members of the community and without them the project would have no purpose. Anyone can be a stakeholder; there are no special requirements.
+
+Stakeholder participation is on a voluntary basis, but the project asks its stakeholders to participate in the project and community as much as possible. Stakeholder contributions enable the project team to ensure that they are satisfying the requirements of those stakeholders. Common stakeholder contributions include (but are not limited to):
+* Evangelism of the project (e.g. a link on a website and word-of-mouth awareness)
+* Informing developers of strengths and weaknesses from a new stakeholder perspective
+* Providing moral support (a ‘thank you’ goes a long way)
+* Providing financial support (the project is open source, but capacity is a constant struggle)
```

Comment:
How does this happen?  If there is not a clear process for providing financial support, then this should probably be removed.